### ISSUE TITLE
Lift the contents of frontend/src/jsonld into a standalone NPM package

### ISSUE THREAD

**Issue by jgonggrijp** [UUDigitalHumanitieslab/readit-interface#95]
Excluding the `ns` module, which is entirely specific to our project. This should wait until these modules have matured, ideally after #91, #92, #93, #94 have been realised and we have confirmed that the frontend code works with #81, but should *not* be postponed until eternity (because that would be wasting a potentially very useful future library).
